test/interpreter/function-hypgeom.dist.spec.ts

```diff
@@ -0,0 +1,98 @@
+import {HyperFormula} from '../../src'
+import {ErrorType} from '../../src/Cell'
+import {ErrorMessage} from '../../src/error-message'
+import {adr, detailedError} from '../testUtils'
+
+describe('Function HYPGEOM.DIST', () => {
+  //In product #1, function takes 4 arguments.
```

Comment:
WDYT - in such situations the note above each different test would be too much clutter? The comment at the top can be easily missed